### PR TITLE
Map changes, rebalances, and fixes.

### PR DESCRIPTION
**Pull request by Kazkin**
-The spider queen and emperor spider now have updated sprites. The spider queen now is more inline with existing spiders (dark color, 8 eyes) and the emperor spider now has a skittering movement animation.
-All stimulant syringes have had their descriptions improved, removing eris lore references for soj lore and fixing some russian-english errors.
-Haloperidol, an improved version of dylovene, now removes all forms of toxins and stimulants instead of just spider venom at double its former rate. **Use this to treat midwife, spider queen, and emperor spider bites. It will knock the person out but rapidly filter unstable mutagen, pararien, menace, and party drops out.**
-Cindarite racial for purge toxins now also filters stimulant at the same rate it does toxins (half what haloperidol can do).
-Midwife spider meat now has unstable mutagen, all spiders now carry their venom type in their chemicals, along with additional chemicals. Thank @Fleshbound#3710 for this.
-The plasma tag trophy is now mapped in and an oddity that gives a bonus to all stats at a moderate level. It is currently held by the church and its position will change after each tourney.
-All laser tag equipment is now properly called plasma tag.
-Lots of minor map changes to pretty things up and improve the colony based on community suggestion in #map-suggestion
-Elite mercs had their speed nerfed to 2 (down from 6) due to being far too speedy.

## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: code/game/objects/items/oddities.dm [hunk 1]
```diff
@@ -266,3 +266,16 @@
 		STAT_ROB = 6,
 		STAT_VIG = 6,
 	)
+
+/obj/item/weapon/oddity/rare/golden_cup
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/oddities.dm:270]: Maybe have this not be-able to get annomlies, so it cant be like `Oh wow their was fungus in the cup...`
just a suggestion

File: code/game/objects/structures/crates_lockers/closets/fitness.dm [hunk 2]
```diff
@@ -54,7 +54,7 @@
 	new /obj/item/clothing/suit/fluff/redtag(src)
 
 /obj/structure/closet/lasertag/blue
-	name = "blue laser tag equipment"
+	name = "blue plasma tag equipment"
 	desc = "It's a storage unit for laser tag equipment."
```
Comment (Trilbyspaceclone) [inline code/game/objects/structures/crates_lockers/closets/fitness.dm:58]:
```suggestion
	desc = "It's a storage unit for plasma tag equipment."
```

File: code/game/objects/structures/crates_lockers/closets/fitness.dm [hunk 3]
```diff
@@ -69,7 +69,7 @@
 	new /obj/item/clothing/suit/fluff/bluetag(src)
 
 /obj/structure/closet/lasertag/green
-	name = "green laser tag equipment"
+	name = "green plasma tag equipment"
 	desc = "It's a storage unit for laser tag equipment."
```
Comment (Trilbyspaceclone) [inline code/game/objects/structures/crates_lockers/closets/fitness.dm:73]:
```suggestion
	desc = "It's a storage unit for plasma tag equipment."
```

File: code/game/objects/structures/crates_lockers/closets/fitness.dm [hunk 4]
```diff
@@ -84,7 +84,7 @@
 	new /obj/item/clothing/suit/fluff/greentag(src)
 
 /obj/structure/closet/lasertag/yellow
-	name = "yellow laser tag equipment"
+	name = "yellow plasma tag equipment"
 	desc = "It's a storage unit for laser tag equipment."
```
Comment (Trilbyspaceclone) [inline code/game/objects/structures/crates_lockers/closets/fitness.dm:88]:
```suggestion
	desc = "It's a storage unit for plasma tag equipment."
```

File: code/modules/mob/living/simple_animal/hostile/syndicate.dm [hunk 5]
```diff
@@ -238,7 +238,7 @@
 	response_help = "pokes"
 	response_disarm = "shoves"
 	response_harm = "hits"
-	speed = 6
+	speed = 2
 	move_to_delay = 1
 	turns_per_move = 1
```
Comment (Trilbyspaceclone) [inline code/modules/mob/living/simple_animal/hostile/syndicate.dm:243]:
```suggestion
```

File: code/modules/reagents/reagents/medicine.dm [hunk 6]
```diff
@@ -864,9 +864,12 @@ datum/reagent/medicine/respirodaxon/affect_blood(var/mob/living/carbon/M, var/al
...
+			var/datum/reagent/toxin/R = current
 			if(!istype(R, src))
-				R.remove_self(effect_multiplier * pick(list(1,2,3)))
+				R.remove_self(effect_multiplier * pick(list(2,4,6)))
+			var/datum/reagent/stim/S = current
+			if(!istype(S, src))
+				S.remove_self(effect_multiplier * pick(list(2,4,6)))
 	var/effective_dose = dose
 	if(issmall(M))
 		effective_dose *= 2
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/reagents/medicine.dm:875]: line 882 shows "else if(effective_dose < 3)"
perhaps make it so its above 11?
Atm is a really really affective as a sleeping chem and will likely be used in 5-10u doses. Meaning its going to sleep a lot of people and be annoying to use even in controlled areas like medical

**Review by Trilbyspaceclone — CHANGES_REQUESTED** [review]
Mostly suggestions and tweaks

File: code/modules/mob/living/simple_animal/hostile/syndicate.dm (hunk 5 above)
Comment (Trilbyspaceclone) [reply code/modules/mob/living/simple_animal/hostile/syndicate.dm:243]: Oh and reason is that its defined for 5 just at line 237 iirc

Comment (Kazkin) [reply code/modules/mob/living/simple_animal/hostile/syndicate.dm:243]: That makes them too slow.

File: code/game/objects/items/oddities.dm (hunk 1 above)
Comment (Kazkin) [reply code/game/objects/items/oddities.dm:270]: Gold cup lottery.

File: code/modules/reagents/reagents/medicine.dm (hunk 6 above)
Comment (Kazkin) [reply code/modules/reagents/reagents/medicine.dm:875]: I like it at 3

File: code/modules/mob/living/simple_animal/hostile/syndicate.dm (hunk 5 above)
Comment (Trilbyspaceclone) [reply code/modules/mob/living/simple_animal/hostile/syndicate.dm:243]: then maybe remove the turns_per_move = 5 var at 237?

**Pull request merged**